src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,88 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.exception.WrongValidationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private RegistrationService registrationService;
+    private User newUser;
+
+    @BeforeEach
+    void setUp() {
+        newUser = new User("ElonMusk99", "eLONmUSK((", 24);
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @AfterEach
+    void clearDataBase() {
+        Storage.PEOPLE.clear();
+    }
+
+    @Test
+    void register_validUser_Ok() {
+        User registratedUser = registrationService.register(newUser);
+        assertEquals(1, Storage.PEOPLE.size());
+        assertEquals(newUser, registratedUser);
+    }
+
+    @Test
+    void register_NullUser_NotOk() {
```

Comment:
Let's stick to one style of naming. Pay attention to Upper letter. Check all test methods.
For example, 
register_nullUser_notOk or register_NullUser_NotOk